### PR TITLE
Add team profiles

### PR DESCRIPTION
**Pull request by Idadelveloper**
This tackles https://github.com/unicef/inventory-hugo-theme/issues/13
@jwflory I will have to probably write some content/tutorial for the various startups outlining how they will include their company on the inventory site.

### CODE REVIEW

**Comment by jwflory** [comment]
I rebased the commits in this Pull Request on top of latest changes to `main`. Local checkouts need to take the following steps to synchronize the new commit history:

```sh
# stash any uncommitted changes so they are not overwritten
git stash

# fetch new commit history from remote
git fetch <remote>

# reset the local branch with the branch in the remote
git reset --hard <remote> team-profiles
```

File: layouts/partials/cohorts.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+            {{range .Site.Home.Sections.ByWeight}}
+              {{ if eq .FirstSection $currentNode.FirstSection }}
+              {{ template "section-tree-nav" dict "sect" . "currentnode" $currentNode}}
+              {{ end }}
+            {{ end }}
+            {{ if .Content }}
+            <div>
+                <button class="btn btn-primary btn-block" id="generatePDF" > Get Pdf  </button>
+            </div>
+            {{ end }}
```
[the remark this answers is not in the document]
Comment (Idadelveloper) [reply layouts/partials/cohorts.html:20]: @jwflory this will be a different PR in the themes repo

**Comment by Idadelveloper** [comment]
> I rebased the commits in this Pull Request on top of latest changes to `main`. Local checkouts need to take the following steps to synchronize the new commit history:
> 
> ```shell
> # stash any uncommitted changes so they are not overwritten
> git stash
> 
> # fetch new commit history from remote
> git fetch <remote>
> 
> # reset the local branch with the branch in the remote
> git reset --hard <remote> team-profiles
> ```

Opps! Already committed some changes @jwflory

File: config.toml [hunk 2]
```diff
@@ -104,6 +104,50 @@ hasChildren = true
...
+    parent = "cohorts"
+    name = "Blockchain"
+    url = "cohorts/blockchain"
+    weight = 20
+
+  [[menu.main]]
+    parent = "cohorts"
+    name = "Drones"
+    url = "cohorts/drones"
+    weight = 30
```
[the remark this answers is not in the document]
Comment (jwflory) [reply config.toml]: Resolved in commit 54e8738b03bd66b09f6f8e39329a505fc4375e07.

File: config.toml [hunk 3]
```diff
@@ -104,6 +104,50 @@ hasChildren = true
...
+    parent = "cohorts"
+    name = "Data Science & AI"
+    url = "cohorts/dsai"
+    weight = 30
+
+  [[menu.main]]
+    parent = "cohorts"
+    name = "VR + AR"
+    url = "cohorts/vr+ar"
+    weight = 30
```
[the remark this answers is not in the document]
Comment (jwflory) [reply config.toml]: Resolved in commit 54e8738b03bd66b09f6f8e39329a505fc4375e07.

File: config.toml [hunk 4]
```diff
@@ -104,6 +104,50 @@ hasChildren = true
...
+
+  [[menu.main]]
+    parent = "cohorts"
+    name = "Others"
+    url = "cohorts/others"
+    weight = 30
```
[the remark this answers is not in the document]
Comment (jwflory) [reply config.toml]: Resolved in commit 54e8738b03bd66b09f6f8e39329a505fc4375e07.

File: content/cohorts/_index.en.adoc [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+title: "Cohorts"
+icon: "vr-cardboard"
+description: "UNICEF’s Office of Innovation uses technology to address the biggest challenges facing children today."
```
[the remark this answers is not in the document]
Comment (jwflory) [reply content/cohorts/_index.en.adoc:4]: @Idadelveloper See this feedback on the cohort index page. ^^

File: content/cohorts/other/_index.en.adoc [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+title: "Others"
+icon: "shapes"
+description: "Develop a more ethical, transparent, and safe AI system with these resources."
```
[the remark this answers is not in the document]
Comment (jwflory) [reply content/cohorts/other/_index.en.adoc:4]: @Idadelveloper See this feedback on the _Other cohorts_ index page. ^^

File: layouts/partials/cohorts.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwflory) [reply layouts/partials/cohorts.html:20]: @Idadelveloper Okay. I opened unicef/inventory-hugo-theme#31 to track this. For now, we can move forward with hard-coding this into the cohorts layout.

**Comment by jwflory** [comment]
@Idadelveloper Your changes look good! 👍🏻 There are a few more things to add in before we merge this, but it is almost there. Once this is merged, we can follow up with updates to the start-up companies to fill their own profiles. 😁 

Note that I pushed a couple of new commits and rebased on latest changes in `main`, so you will need to update your local git branch from the remote again before iterating further.

**Comment by Idadelveloper** [comment]
@jwflory I think it's good now. Just rebased the changes

File: layouts/partials/cohorts.html [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+              <p>Below are the DPG indicators {{.Params.Title}} has met:</p>
+              <ul>
+                {{ range .Params.dpgs }}
+                  {{ $num:= . }}
+                  {{ $base:= "dpg" }}
+                  {{ $dpg:= printf "%s" $num | printf "%s%s" $base | printf "%s" }}
+                  {{ range $dpg_i := $.Site.Params.dpgs }}
+                    {{ range $key, $value := $dpg_i }}
+                      {{ if eq $key $dpg }}
+                        <li><i class="fa fa-check mr-2" aria-hidden="true"></i>{{ $value }}</li>
```
Comment (jwflory) [inline layouts/partials/cohorts.html:50]: This is slick! 😏

File: layouts/partials/cohorts.html [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+              <p>{{ .Content }}</p>
+             </div>
+             <div>
+              <h2>DPG Indicators</h2>
+              <p>The Digital Public Goods Standard is a set of specifications and guidelines designed to maximise consensus about whether a digital solution conforms to the definition set by the UN Secretary-General in the 2020 Roadmap for Digital Cooperation, which states that a DPG, “is open source software, open data, open AI models, open standards and open content that adhere to privacy and other applicable best practices, do no harm and are of high relevance for attainment of the UN’s 2030 Sustainable Development Goals (SDGs)”. Learn more about the DPG <a href="https://digitalpublicgoods.net/standard/">here</a>.</p>
+              <p>Below are the DPG indicators {{.Params.Title}} has met:</p>
+              <ul>
+                {{ range .Params.dpgs }}
+                  <li><i class="fa fa-check mr-2" aria-hidden="true"></i>{{ . }}</li>
+                {{ end }}
```
[the remark this answers is not in the document]
Comment (jwflory) [reply layouts/partials/cohorts.html]: This is updated to use integers over strings, nice work @Idadelveloper 👍🏻

**Comment by jwflory** [comment]
Hey @Idadelveloper, did you catch the changes for the [cohort index page](https://github.com/unicef/inventory/pull/78/files/3159f1351578c20206b2d506a449fc10ab6a3ad2#diff-9f449835a13f92ab00a7e1286065bf2b40d325501fb1cdb1e61627aed8df7134) and [other cohorts index page](https://github.com/unicef/inventory/pull/78/files/3159f1351578c20206b2d506a449fc10ab6a3ad2#diff-1e25572887444a9019d8b2309ce81ea6c7f298b37b006d4a7123047620184f0a)?

File: content/cohorts/_index.en.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (jwflory) [reply content/cohorts/_index.en.adoc]: Resolved in commit 48f520ecc83557fc5cf484d6bc925748f3896b36.

File: content/cohorts/other/_index.en.adoc (hunk 6 above)
[the remark this answers is not in the document]
Comment (jwflory) [reply content/cohorts/other/_index.en.adoc]: Resolved in commit 48f520ecc83557fc5cf484d6bc925748f3896b36.

**Review by jwflory — APPROVED** [review]
After much ado, this is ready to merge! 😀 Super work on getting this implemented @Idadelveloper! 🎉  I am super excited for us to start leveraging this feature with the start-up companies. Merging! 🎬 

And a special thanks to @Erioldoesdesign for helping out with the design-thinking part of this feature to bring it to life. 🌟

**Pull request merged**